### PR TITLE
chore, refactor(egen): follows new template, removes IS_TESTING, spell corrections

### PR DESCRIPTION
**Pull request by krishr2d2**
<br>
This PR includes the following updates to the `get_started_with_model_monitoring_custom.ipynb` notebook:

- Replace K80 with T4
- Move steps for deleting the endpoints to cleaning up section
- Adds Colab steps from the new template
- Fix/reword sentences to explain some steps in a better and simpler way
- Spelling/typo corrections
- Remove unnecessary code highlights (``)
- Add Colab Enterprise link and other steps from the new template
- Remove future tense
- Contract words like is not, do not etc.
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

### CODE REVIEW

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_custom.ipynb [hunk 1]
```diff
@@ -572,26 +549,26 @@
...
         "\n",
-        "For skew detection, the monitoring service requires a baseline for the statistical distribution of values in the training data. For AutoML tabular models this is automatically derived. For custom tabular models, you upload the training data to the service, and have the service automatically derive the distribution.\n",
+        "For skewness detection, the monitoring service requires a baseline for the statistical distribution of values in the training data. For AutoML tabular models this is automatically derived. For custom tabular models, you upload the training data to the service, and have the service automatically derive the distribution.\n",
         "\n",
-        "For feature attribution skew and drift detection, requires enabling your deployed model for `Vertex AI Explainability` for custom tabular models. For AutoML models, `Vertex AI Explainability` is automatically enabled.\n",
+        "For skewness and drift detection in feature attributions, you're required to enable Vertex AI Explainability feature for your deployed custom tabular models. For AutoML models, Vertex AI Explainability is automatically enabled.\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_custom.ipynb:571]: Change:

- Vertex AI Explainability

To:

- Vertex Explainable AI

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_custom.ipynb [hunk 2]
```diff
@@ -962,17 +939,17 @@
       "source": [
         "### Create the monitoring job\n",
         "\n",
-        "You create a monitoring job, with your monitoring specifications, using the `aiplatform.ModelDeploymentMonitoringJob.create()` method, with the following parameters:\n",
+        "Create a monitoring job, with your monitoring specifications, using Vertex AI's `ModelDeploymentMonitoringJob.create()` method, with the following parameters:\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_custom.ipynb:942]: perhaps add a link to the `ModelDeploymentMonitoringJob.create()` method:

https://cloud.google.com/python/docs/reference/aiplatform/1.48.0/summary_method#google_cloud_aiplatform_ModelDeploymentMonitoringJob_create_summary

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_custom.ipynb [hunk 3]
```diff
@@ -1182,17 +1159,17 @@
...
         "\n",
-        "Once the analysis is completed, the monitoring job will send email notifications on the detected skew, in this case `country`, and the monitoring job will go into `OFFLINE` state until the next interval.\n",
+        "Once the analysis is completed, the monitoring job sends email notifications on the detected skewness, in this case `country`. Further, the monitoring job goes into **OFFLINE** state until the next interval.\n",
         "\n",
         "#### Wait for monitoring interval\n",
         "\n",
-        "It can take upwards of 40 minutes from when the analyis occurred on the monitoring interval to when you receive an email alert.\n",
+        "It can take upto 40 minutes or more from the moment the analysis on the monitoring interval is done until you receive an email alert.\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_custom.ipynb:1170]: change "upto" to "up to" (two words)

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_custom.ipynb [hunk 4]
```diff
@@ -1347,13 +1324,13 @@
...
         "\n",
-        "Once the analysis is completed, the monitoring job will send email notifications on the detected drift, in this case `cnt_user_engagement`, and the monitoring job will go into `OFFLINE` state until the next interval.\n",
+        "Once the analysis is completed, the monitoring job sends email notifications on the detected drift, in this case `cnt_user_engagement`. Then, the monitoring job goes into **OFFLINE** state until the next interval.\n",
         "\n",
         "#### Wait for monitoring interval\n",
         "\n",
-        "It can take upwards of 40 minutes from when the analyis occurred on the monitoring interval to when you receive an email alert."
+        "It can take upto 40 minutes or more from the moment the analyis on the monitoring interval is done until you receive an email alert."
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_custom.ipynb:1333]: change "upto" to "up to" (two words)

**Review by kittyabs — COMMENTED** [review]
suggested edits

**Review by katiemn — APPROVED** [review]

**Pull request merged**